src/coinbase/wallet.ts

```diff
@@ -778,7 +779,7 @@ export class Wallet {
    * @param options.symbol - The symbol of the ERC20 token.
    * @param options.totalSupply - The total supply of the ERC20 token.
    * @returns A Promise that resolves to the deployed SmartContract object.
-   * @throws {Error} If the private key is not loaded when not using server signer.
+   * @throws {APIError} If the private key is not loaded when not using server signer.
```

Comment:
I believe this throws an `Error` if the private key is not loaded when not using server signer and an `APIError` if the backend returns an error.

Reply:
ooh good call! i blindly changed it - fixed now